test/features/homepage.feature

```diff
@@ -1,6 +0,0 @@
-@smoke
```

Comment:
as in other repos - please revert this . it is required to make sure that all systems are working correctly

Reply:
homepage.feature smoke test re-instated.